packages/reaction-product-variant/client/helpers/global.js

```diff
@@ -71,21 +72,33 @@ ReactionProduct.publishProduct = function (productOrArray) {
 ReactionProduct.cloneProduct = function (productOrArray) {
   const products = !_.isArray(productOrArray) ? [productOrArray] : productOrArray;
 
-  return Meteor.call("products/cloneProduct", products, function (error) {
+  return Meteor.call("products/cloneProduct", products, function (error, result) {
     if (error) {
+      Alerts.add(error, "danger", { placement: "productGridItem" });
       throw new Meteor.Error("error cloning product", error);
     }
-    for (let product of products) {
-      Alerts.add(i18next.t("productDetail.clonedAlert") + " " + product.title, "success", {
-        placement: "productGridItem",
-        id: product._id,
-        autoHide: true,
-        dismissable: false
-      });
+    if (result) {
+      if (products.length === 1) {
+        Alerts.add(i18next.t("productDetail.clonedAlert", { product: products[0].title }), "success", {
+          placement: "productGridItem",
+          id: products[0]._id,
+          autoHide: true,
+          dismissable: false
+        });
+      } else {
+        Alerts.add(i18next.t("productDetail.clonedAlert_plural", { product: i18next.t("productDetail.theSelectedProducts"), count: 0 }),
+          "success", {
+            placement: "productGridItem",
+            id: products[0]._id,
+            autoHide: true,
+            dismissable: false
+          }
+        );
+      }
     }
-    if (!_.isArray(productOrArray)) {
+    if (products.length === 1) {
```

Comment:
As far as I understand this logic was prepared for a future usage from PDP? Because currently we don't have cloning functionality within PDP. I'll revert it asap.